react-version/src/components/game/buttons/ButtonsRow9.js

```diff
@@ -1,6 +1,15 @@
-import React from "react";
+import React, { useContext } from "react";
+import { BoardContext } from "../../context/BoardContext";
 
 const ButtonsRow9 = (props) => {
+	const [board, setBoard] = useContext(BoardContext);
+	const handleBoardBtn = (e) => {
```

Comment:
You see now how many code is duplicates over and over if we have those buttons rows? :)